pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ['setuptools>=40']
```

Comment:
For what specifically is this file needed?

Reply:
It's used to configure setuptools with [pyproject.toml](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html), but it's not necessary.